e2e/components/ActionList.test.ts

```diff
@@ -478,4 +478,45 @@ test.describe('ActionList', () => {
       })
     }
   })
+
+  test.describe('All combinations', () => {
+    for (const theme of themes) {
+      test.describe(theme, () => {
+        test('default @vrt', async ({page}) => {
+          await visit(page, {
+            id: 'components-actionlist-examples--all-combinations',
+            globals: {
+              colorScheme: theme,
+            },
+          })
+
+          // Resize to fit long page
+          await page.setViewportSize({width: 1000, height: 1000})
+
+          // Default state
+          expect(await page.screenshot()).toMatchSnapshot(`ActionList.All combinations.${theme}.png`)
+
+          // Hover state
+          await page.getByRole('listitem', {name: 'Danger inline description'}).hover()
+          expect(await page.screenshot()).toMatchSnapshot(`ActionList.All combinations.${theme}.hover.png`)
+        })
+
+        test('axe @aat', async ({page}) => {
+          await visit(page, {
+            id: 'components-actionlist-features--all-combinations',
+            globals: {
+              colorScheme: theme,
+            },
+          })
+          await expect(page).toHaveNoViolations({
+            rules: {
+              'color-contrast': {
+                enabled: theme !== 'dark_dimmed',
+              },
+            },
+          })
+        })
```

Comment:
This should help out with the `@aat` rules, we ended up updating the defaults for contrast checks while the primitive work is underway 👍 

```suggestion
          await expect(page).toHaveNoViolations()
```